index.html

```diff
@@ -1008,41 +1001,30 @@
 }
       </pre>
     </section>
-    <section data-dfn-for="PaymentDetails" data-link-for="PaymentDetails">
+    <section data-dfn-for="PaymentDetailsBase" data-link-for=
```

Comment:
It might be nice to nest these three sections under a single section (e.g. "Payment details dictionaries")